### ISSUE TITLE
the map show nothing in release build, normal in debug build

### ISSUE THREAD

**Issue by vjerryv** [ashare80/TSClusterMapView#37]
I tested all and everything well in debug build, but when I upload to app store, try on test flight.
It is nothing to show on the map.

version 2.3.1

**Comment by AdamShare** [comment]
@vjerryv Interesting.  Is there a pervious version that works properly for you?

**Comment by vjerryv** [comment]
2.1.0 works

**Comment by AdamShare** [comment]
Hmm ok I merged in a branch that added support for OSX so if 2.1.0 works as needed on iOS I would just stick with that version for now. I'll let you know if I find the issue and update.

**Comment by vjerryv** [comment]
thanks

**Comment by appfabtech** [comment]
I can confirm the problem. With release build you get the first redraw of the clusters then nothing happens, on zoom, on data refresh...

**Comment by appfabtech** [comment]
@vjerryv don't you have any issue with map refresh using 2.1.0? I rolled back at that version and not when I replace the annotations set (after a new textual search that change the dataset for the map) the new annotations (singles and clustered) are not visible until I manual zoom in or out the map.

**Comment by vjerryv** [comment]
Yes, there are many critical problem while using TSClusterMapView.

**Comment by appfabtech** [comment]
Hello @ashare80, any chance to have the issue fixed?
thank you :)

**Comment by AdamShare** [comment]
Sorry super busy. Can't seem to reproduce the issue, is this still a problem on the latest update?
